cscs-checks/microbenchmarks/spec-accel/spec.py

```diff
@@ -0,0 +1,121 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+from reframe.core.launchers.registry import getlauncher
+
+
+class SpecAccelCheckBase(rfm.RegressionTest):
+    def __init__(self, prg_envs):
+        super().__init__()
+        self.valid_systems = ['daint:gpu', 'dom:gpu']
+        self.valid_prog_environs = prg_envs
+        self.modules = ['craype-accel-nvidia60']
+
+        self.configs = {
+            'PrgEnv-gnu':  'cscs-gnu',
+            'PrgEnv-cray': 'cscs-cray',
+        }
+
+        app_source = os.path.join(self.current_system.resourcesdir,
+                                  'SPEC_ACCELv1.2')
+        self.prebuild_cmd = ['cp -r %s/* .' % app_source,
+                             './install.sh -d . -f']
+
+        # I just want prebuild_cmd, but no action for the build_system
+        # is not supported, so I find it something useless to do
+        self.build_system = 'SingleSource'
+        self.sourcepath = './benchspec/ACCEL/353.clvrleaf/src/timer_c.c'
+        self.build_system.cflags = ['-c']
+
+        self.refs = {
+            env: {bench_name: (rt, None, 0.1, 'Seconds')
+                  for (bench_name, rt) in
+                  zip(self.benchmarks, self.runtimes[env])}
+            for env in self.valid_prog_environs
+        }
+
+        self.num_tasks = 12
+        self.num_tasks_per_node = 12
```

Comment:
Why do you need 12 tasks per node?

Reply:
Just wanted to make sure the whole node is available to the test. Changed to 1 task.